pressbooks/pressbooks-book-directory-fe
# chore: bump search-insights library

Issue #826 

This PR bumps the search-insights library and apply the necessary upgrade changes.

**How to test**

1. Checkout this branch
2. Run `npm install` to get the latest version of `search-insights` library
3. Run Pressbooks Directory
4. In the directory, apply filters, search parameters and click around
5. You should see events being sent in the Events page in Algolia dashboard at https://www.algolia.com/apps/{APP_ID}/events/hub
